src/app/AttributePathParams.h

```diff
@@ -18,6 +18,7 @@
 
 #pragma once
 
+#include <app/AppBuildConfig.h>
```

Comment:
But there are other files that check CHIP_CONFIG_ENABLE_READ_CLIENT but don't include this header, no?

Reply:
I have added the following code in the files that uses `CHIP_CONFIG_ENABLE_READ_CLIENT`
```
#if !undefined(CHIP_CONFIG_ENABLE_READ_CLIENT)
#error "CHIP_CONFIG_ENABLE_READ_CLIENT undefined"
#endif
```
Only this file reports this error.

Reply:
Looks like other files includes some headers that include `app/AppBuildConfig.h`.

Reply:
> Looks like other files includes some headers that include app/AppBuildConfig.h.

That's not reliable.  Those headers can change.  If those files depend on `AppBuildConfig.h`, those files should include it directly.

Reply:
Added this header for all the other files